tools/test-ci.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd `dirname $0`/.. > /dev/null
+if [ -z ${PYTHON+x} ]; then export PYTHON=`which python`; fi
```

Comment:
Yes, but I think setting `PYTHON=''` should give the default behaviour. Also, setting `PYTHON=python` in the `then` case here should be just fine.